flight/odlc_search_grid/search_path.py

```diff
@@ -0,0 +1,86 @@
+"""
+Functions for generating search paths to cover an area for finding the standard odlc objects
+"""
+
+from typing import List, Dict, Tuple
+from shapely.geometry import Polygon
+import utm
+
+
+def latlon_to_utm(coords: Dict[str, float]) -> Dict[str, float]:
+    """Converts latlon coordinates to utm coordinates and adds the data to the dictionary
+
+    Parameters
+    ----------
+    coords : Dict[str, float]
+        A dictionary containing lat long coordinates
+
+    Returns
+    -------
+    Dict[str, float]
+        An updated dictionary with additional keys and values with utm data
+    """
+
+    utm_coords = utm.from_latlon(coords["latitude"], coords["longitude"])
```

Comment:
Please have a type hint for this new structure, stating if its a list, what it contains, etc.